pkg/routers/api.go

```diff
@@ -135,6 +138,7 @@ func WithAPIRouter(opt *APIOptions, r chi.Router) {
 		r.With(opt.oAuthMiddleware).Get("/notifications/event-stream", opt.nStreamHandler)
 	})
 
+	r.With(healthTimer).Get("/health", opt.healthHandler)
```

Comment:
We also need to make this endpoint configurable from within `config.go`. This applies to the api listener as well as the admin listener.